File: PHP Word Examples/EncryptApiDemo/encryptDocumentInRequest.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\EncryptApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new EncryptApi($configuration);

$filename = "D:/data/Template.docx";
$old_password = null;
$new_password = '123456';
$result = $apiInstance->encryptDocumentInRequest($filename, $old_password, $new_password);

File: Python Word Examples/EncryptApiDemo/encryptDocumentInRequest.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.encrypt_api.EncryptApi(configuration)

file_name = 'EncryptDocumentInRquest.docx'
input_file = "D:/input/"
extra = {'new_password': '123456'}
result = api.encrypt_document_in_request(input_file, **extra)